index.html

```diff
@@ -606,6 +609,12 @@ <h4>Suppressing a pointer event stream</h4>
                 </section>
             </section>
 
+            <section>
+                <h3><dfn data-lt="events-from-layout-changes">Boundary events caused by layout changes</dfn></h3>
+                <p>A pointing device that moved relative to the screen surface or underwent some change in any of its properties fires various events as defined in <a>Pointer Event types</a>.  For a stationary pointing device (that neither moved relative to the screen surface nor underwent any change in any properties), the user agent MUST fire certain boundary events after a layout change that affected the <a>hit test</a> target for the pointer, see {{GlobalEventHandlers/pointerover}}, {{GlobalEventHandlers/pointerenter}}, {{GlobalEventHandlers/pointerout}} and {{GlobalEventHandlers/pointerleave}} for details.  The user agent MAY delay the firing of these boundary events because of performance reasons (e.g. to avoid too many hit-tests or layout changes caused by boundary event listeners).</p>
```

Comment:
> The user agent MAY delay the firing of these boundary events because of performance reasons (e.g. to avoid too many hit-tests or layout changes caused by boundary event listeners).

I think it should be defined here that if the last `pointerover` target appears under the pointer after temporarily removed. I think `pointerover` should be fired again to avoid the timing issue of the delay vs. async handling in web apps.  Otherwise, we could see another issue which is the case when the last `pointerover` target is back under the pointer but some ancestor elements which got `pointerenter` events fired are not back again.

(It's fine to handle this in a new issue.)

Reply:
I totally agree this PR is unclear about such DOM modifications.  Let's fix it separately after this PR lands.

I think the proper way to address such cases would be introducing a state variable for "current element under pointer" for each pointer, and dispatch boundary events whenever this changes.  That will also help us port the [Mouse Event Order](https://w3c.github.io/uievents/#events-mouseevent-event-order) section from UIEvent into PoitnerEvent spec (which will nicely cover a part of our TPAC 2024 plan).